tools/c7n_azure/c7n_azure/resources/monitor_logprofile.py

```diff
@@ -0,0 +1,40 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from c7n_azure.provider import resources
+from c7n_azure.resources.arm import ArmResourceManager
+
+
+@resources.register('monitor-logprofile')
```

Comment:
it looks fine but wondering about the consistency/inconsistency on the resource name wrt to hyphenation.

Reply:
@kapilt Thx, That was an oversight, I changed the resource name to 'monitor-log-profile'.